### PR TITLE
Problem using android-cucumber with newer version of the android-maven-plugin

### PR DESCRIPTION
**Pull request by brasmusson**
Newer versions of the android-maven-plugin does not seem to allow apklib:s to be included when building an instrumentation test debug apk.

With android-maven-plugin v4.0.0-rc.1 (and AFAIK v3.9.0-rc3), the generate source phase for a project that depend on cucumber-android will fails since cucumber-android is not extracted ([see Travis build](https://travis-ci.org/cucumber/cucumber-jvm/jobs/39746939)):
```
[INFO] --- android-maven-plugin:4.0.0-rc.1:generate-sources (default-generate-sources) @ cucumber-android-test ---
[INFO] Extracting libs
[ERROR] Error when generating sources.
org.apache.maven.plugin.MojoExecutionException: AndroidManifest.xml file wasn't found in next place:  /home/travis/build/cucumber/cucumber-jvm/examples/android/android-test/target/unpacked-libs/ic_cucumber-android
```
With android-maven-plugin v3.9.0-rc.1 cucumber-android is extracted, and its classes are included in the built artefact ([see Travis build](https://travis-ci.org/cucumber/cucumber-jvm/jobs/39747984)):
```
[INFO] --- android-maven-plugin:3.9.0-rc.1:generate-sources (default-generate-sources) @ cucumber-android-test ---
[INFO] Manifest copying disabled. Using default manifest only
[INFO] Extracting libs
[INFO] Extracting apklib cucumber-android...
```

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ matrix:
     script: mvn -q deploy --settings .travis-settings.xml -Dno.gem.deploy=true
   - jdk: oraclejdk7
     env: ANDROID=true
-    script: mvn -q deploy -P android -pl android --settings .travis-settings.xml -Dno.gem.deploy=true -Dandroid.device=test
+    script: mvn install -P android,android-examples -pl android,examples/android -amd --settings .travis-settings.xml -Dno.gem.deploy=true -Dandroid.device=test && mvn -q deploy -P android -pl android --settings .travis-settings.xml -Dno.gem.deploy=true -Dandroid.device=test
```
Comment (SierraGolf) [inline .travis.yml]: we probably also want to run the android studio / gradle example here.

**Comment by SierraGolf** [comment]
@brasmusson do we want to recycle this pull request to become the pull request which adds ci support for the android examples or do we want to close it and create a new one?

**Comment by brasmusson** [comment]
@SierraGolf Let's close this PR and create a new for the ci support for the android examples.

**Comment by brasmusson** [comment]
Fixed by #819.

**Pull request closed without merging**